src/addrman.cpp

```diff
@@ -854,6 +854,7 @@ std::vector<std::pair<AddrInfo, AddressPosition>> AddrManImpl::GetEntries_(bool
             int id = GetEntry(from_tried, bucket, position);
             if (id >= 0) {
                 AddrInfo info = mapInfo.at(id);
+                info.mapped_as = m_netgroupman.GetMappedAS(info);
```

Comment:
Maybe cleaner to set this when AddrInfo is first initialized?

Could also get rid of two `const auto mapped_as{m_netgroupman.GetMappedAS(addr)};` in addrman.cpp this way

Reply:
Another thought: We can't do this - someone else might see: "oh nice `mapped_as`" is already a field" and will use it, but it hasn't been set because nobody called GetEntries().